src/store/segmentGroups.ts

```diff
@@ -8,11 +8,15 @@ import { useIdStore } from '@/src/store/id';
 import { onImageDeleted } from '@/src/composables/onImageDeleted';
 import { normalizeForStore, removeFromArray } from '@/src/utils';
 import { compareImageSpaces } from '@/src/utils/imageSpace';
-import { LabelMapSegment } from '@/src/types/labelmap';
-import { DEFAULT_LABELMAP_SEGMENTS } from '@/src/config';
+import { SegmentMask } from '@/src/types/segment';
```

Comment:
When should we prefer camelCase over kebab-case for file names?

Reply:
I've been switching to use camelCase for file names, so I'd recommend doing that. It is mostly just a developer cosmetic, so I'm not looking to change all at once, and I'm not going to consider it strongly in a review.